static-site/src/components/splash/index.jsx

```diff
@@ -215,57 +217,163 @@ const UrlShowcaseTile = ({ card }) => {
     <CardOuter>
       <CardInner>
         <a href={card.url}>
-          <CardTitle $squashed>
-            {card.name}
+          <CardTitle>
+              {card.name}
           </CardTitle>
-          <CardImgWrapper filename={card.img}/>
+          <CardImgContainer>
+            <CardImgWrapper filename={card.img}/>
+            <InfoIcons>
+              <CardSourceIcon url={card.url} />
+              {card.type == 'narrative' && <NarrativeIcon />}
+            </InfoIcons>
+          </CardImgContainer>
         </a>
+        <CardDescription>
+          {card.description}
+        </CardDescription>
       </CardInner>
     </CardOuter>
   )
 }
 
-const cardWidthHeight = 160; // pixels
+function CardSourceIcon({ url }) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const gitHubLogo = require(`../../../static/logos/github-mark.png`).default.src;
+
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const nextstrainLogo = require(`../../../static/logos/nextstrain-logo-tiny.png`).default.src;
+
+  let maintainers, image;
+
+  if (url.startsWith('/community')) {
+    maintainers = 'community members on GitHub';
```

Comment:
[suggestion]

Change this text to "{orgName} via GitHub"

Reply:
Good suggestion, added in 694ff93b02a94d1434e990433e67cd6c3a82a7b3